internal/controller/controllers/bmh_agent_controller.go

```diff
@@ -939,6 +956,95 @@ func (r *BMACReconciler) ensureSpokeMachine(ctx context.Context, log logrus.Fiel
 	return machineSpoke, nil
 }
 
+// Get MCS cert of the spoke cluster, encode it and add it to BMH
```

Comment:
It would be good, for future reference, to document here what this function does and why we need it.

I think something along the lines of:

In order for remote worker nodes  to be able to .... we need to pull the MCS certificate from the spoke cluster and inject it as part of the ignition config before the node boots so that ... "

Reply:
Done